### PR TITLE
Slight typo in comments of DOM Methods.

### PR DESCRIPTION
**Pull request by Utkarsh736**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [x] The title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 - [ ] If the PR is related to an open issue, use a [relevant keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) and reference it with the `#` sign and the issue number.
 - [x] You have previewed your markdown (if any) in the [Odin Markdown Preview Tool](https://www.theodinproject.com/lessons/preview)
 - [x] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).

#### 1. Describe the changes made and include why they are necessary or important:

Hey, while going through the different methods when I saw comments explaining the methods,
I got a bit confused with the comment  //select the container div. 

Like does the command selects the div or are we required to select the div ourselves after that command.
I think it should be 'selects' as was in the 4th comment.
I have made similar changes in other comments as well.

#### 2. Related Issue

Closes #XXXXX

### CODE REVIEW

**Comment by twalton83** [comment]
It looks like this is using the imperative voice.

**Comment by Utkarsh736** [comment]
> It looks like this is using the imperative voice.

Thanks for the reply, but could you elaborate on this a bit more.
Also, which one do you refer to with 'this', the original sentence or my suggestion?

**Comment by twalton83** [comment]
Meaning, the comments are written with an imperative voice, and your updates remove that.

**Comment by Utkarsh736** [comment]
Honestly, I'm not good with understanding voices and which to use when.

But I feel it should be uniform in whichever way it is,
Like the first 3 comments in the "Targeting node with selectors" section use the word 'select',
and the 4th comment uses the word 'selects'.
 
Similarly in other comments, it starts one way and changes midway.

The confusion for me personally was,
That I found it a bit confusing as to whether the comment is explaining to me the functioning of the command or if it was telling me to select something after that command.

P.S. Sorry for being an inconvenience but I am new to this, so feel free to comment if I am breaking some protocols or etiquettes.

**Comment by twalton83** [comment]
It’s not an inconvenience, the review process is making comments and observations about the changes, just to be clear.

**Comment by Utkarsh736** [comment]
Oh Ok, Thanks for the clarification.

File: foundations/javascript_basics/DOM_manipulation_and_events.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ div.style.cssText = "background-color: white" // ok in a string
 
 ~~~JavaScript
 div.setAttribute('id', 'theDiv');                              
-// if id exists update it to 'theDiv' else create an id
+// if id exists updates it to 'theDiv' else creates an id
```
Comment (zachmmeyer) [inline foundations/javascript_basics/DOM_manipulation_and_events.md:127]:
```suggestion
// if id exists, update it to 'theDiv', else create an id
```

File: foundations/javascript_basics/DOM_manipulation_and_events.md [hunk 2]
```diff
@@ -144,11 +144,11 @@ div.classList.add('new');
 // adds class "new" to your new div
 
 div.classList.remove('new');                                   
-// remove "new" class from div
+// removes "new" class from div
 
 div.classList.toggle('active');                                
-// if div doesn't have class "active" then add it, or if
-// it does, then remove it
+// if div doesn't have class "active" then adds it, or if
```
Comment (zachmmeyer) [inline foundations/javascript_basics/DOM_manipulation_and_events.md:150]:
```suggestion
// if div doesn't have class "active" then add it, or if
```

File: foundations/javascript_basics/DOM_manipulation_and_events.md [hunk 3]
```diff
@@ -144,11 +144,11 @@ div.classList.add('new');
...
+// it does, then removes it
```
Comment (zachmmeyer) [inline foundations/javascript_basics/DOM_manipulation_and_events.md:151]:
```suggestion
// it does, then remove it
```

**Review by zachmmeyer — CHANGES_REQUESTED** [review]
Looking like we were already using `selects`, the comments here should be consistent and using `selects` seems most correct. Only a few requested changes here.

**Comment by Utkarsh736** [comment]
Hey, @zachmmeyer Thanks for the reply.
Can you tell me what I'm supposed to do here `commit suggestion` or `resolve conversation`?

**Comment by zachmmeyer** [comment]
> Hey, @zachmmeyer Thanks for the reply.
> Can you tell me what I'm supposed to do here `commit suggestion` or `resolve conversation`?

Commit suggestion would make the changes requested.

**Comment by Utkarsh736** [comment]
Thanks,
I've done `commit suggestions` for the 3 suggestions,
Comment if anything else is needed from my side.

**Review by zachmmeyer — APPROVED** [review]
Thanks for contributing and making these changes!

**Pull request merged**